core/templates/components/shared-component.module.ts

```diff
@@ -40,6 +40,9 @@ import { ExplorationEmbedButtonModalComponent } from
 import { SocialButtonsComponent } from
   'components/button-directives/social-buttons.component';
 import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
+import { ExplorationSummaryTileDirective } from
+  './summary-tile/exploration-summary-tile.directive';
+
```

Comment:
Why is this extra line added? Also, I see three line breaks now, it should be one.

Reply:
Done!